archetypes/se/basic/src/main/resources/src/main/java/__pkg__/Main.java.mustache

```diff
@@ -49,83 +40,58 @@ public final class Main {
      * @throws IOException if there are problems reading logging properties
      */
     static WebServer startServer() throws IOException {
-#if( $loggingConfig.matches("y|yes|true") )
         // load logging configuration
         setupLogging();
-#end
 
-#if( $applicationYaml.matches("y|yes|true") )
         // By default this will pick up application.yaml from the classpath
         Config config = Config.create();
-#end
-
-        WebServer server = WebServer.builder()
-#if( $applicationYaml.matches("y|yes|true") )
-            .routing(createRouting(config))
-            .config(config.get("server"))
-#else
-            .routing(createRouting())
-#end
-#if( $jsonSupport.matches("y|yes|true") )
-            .addMediaSupport(JsonpSupport.create())
-#end
-            .build();
+
+        // Get webserver config from the "server" section of application.yaml
+        ServerConfiguration serverConfig =
```

Comment:
This is now deprecated. Please switch to new API (as was in old template in master).